Objects/genericaliasobject.c

```diff
@@ -226,6 +227,25 @@ _Py_make_parameters(PyObject *args)
             return NULL;
         }
         if (subst) {
+            PyObject *default_;
+            bool does_not_contain = tuple_index(parameters, nargs, t) == -1;
+            if (does_not_contain) {
+                if (_PyObject_LookupAttr(t, &_Py_ID(__default__), &default_) < 0) {
+                    Py_DECREF(default_);
```

Comment:
Isn't default_ uninitialized here? I think we need to DECREF it in the next branch instead.